extensions/stackdriver/stackdriver.cc

```diff
@@ -461,8 +461,8 @@ inline bool StackdriverRootContext::isOutbound() {
   return direction_ == ::Wasm::Common::TrafficDirection::Outbound;
 }
 
-inline bool StackdriverRootContext::enableServerAccessLog() {
-  return !config_.disable_server_access_logging() && !isOutbound();
+inline bool StackdriverRootContext::enableAccessLog() {
+  return !config_.disable_server_access_logging();
```

Comment:
I feel like we want more config around enabling client access logging and, relatedly, we need to assess whether or not `disable_server_access_logging` should be renamed.

Reply:
should we just  add `disable_client_access_logging` ? changing `disable_server_access_logging` might cause compatibility issues?

Reply:
Added `disable_client_access_logging`